apps/angular/projection/src/app/component/city-card/city-card.component.ts

```diff
@@ -1,13 +1,40 @@
-import { Component, OnInit } from '@angular/core';
+import { Component, OnInit, inject } from '@angular/core';
+import { CityStore } from '../../data-access/city.store';
+import {
+  FakeHttpService,
+  randomCity,
+} from '../../data-access/fake-http.service';
+import { City } from '../../model/city.model';
+import { CardComponent } from '../../ui/card/card.component';
 
 @Component({
   selector: 'app-city-card',
-  template: 'TODO City',
+  template: `
+    <app-card
+      [list]="cities"
+      customClass="bg-light-blue"
+      (addNewItem)="store.addOne(randomCity())"
+      (itemDelete)="store.deleteOne($event)">
+      <img src="assets/img/city.png" width="200px" />
+    </app-card>
+  `,
   standalone: true,
-  imports: [],
+
+  imports: [CardComponent],
 })
 export class CityCardComponent implements OnInit {
-  constructor() {}
+  cities: City[] = [];
+  randomCity = randomCity;
+
+  private http = inject(FakeHttpService);
+  protected store = inject(CityStore);
+
+  ngOnInit(): void {
+    this.http.fetchCities$.subscribe((s) => {
+      console.log(s);
+      this.store.addAll(s);
+    });
 
-  ngOnInit(): void {}
+    this.store.cities$.subscribe((s) => (this.cities = s));
```

Comment:
I think you don't have to subscribe and set the `cities` in here. In line 26 just declare cities as `cities = toSignal(this.store.cities$);` or something like that.